components/turret.py

```diff
@@ -91,6 +91,18 @@ def _slew_to_count(self, count: int) -> None:
             self.incrementing = False
         self.current_state = self.SLEWING
 
+    """
+    Slew the turret back and forth looking for a target.
+    """
+
+    def scan(self, heading):
```

Comment:
I think you actually wanted this as this method's docstring, no?

Reply:
Yes. Fixed in latest push.